signer/src/testing/transaction_coordinator.rs

```diff
@@ -287,8 +285,6 @@ where
             .saturating_sub(crate::WITHDRAWAL_BLOCKS_EXPIRY)
             .saturating_add(crate::WITHDRAWAL_EXPIRY_BUFFER);
 
-        // Assert that there are some withdrawals in storage while get_pending_requests return 0 withdrawals
-        assert!(!withdrawals_in_storage.is_empty());
```

Comment:
Why do we want to remove this check?

Reply:
I'm not sure we want, but right now it's still possible for the testdata to generate no "valid" withdrawals (albeit rarely, 1 in ~100 tries). So either we remove this check, or we manually add a withdrawal -- I went for the former, nothing against the latter, I'm not sure if it's significantly better since it would be an "obvious" one.

Reply:
Yeah, makes sense. Any idea for why it's possible to not have any valid withdrawals in the database? I thought that each stacks block had withdrawals in confirmed within them, even the ones not on the canonical chain. Maybe the change is to force consecutive blocks?

Reply:
Hmm, I don't mind to merge adding withdrawals to storage manually or even break our "no flaky tests" rule and merge this. But, then we should open an issue to dig into our testing infra and understand why we don't have withdrawals in storage in this case (and I suspect it is because in our testing helpers we do something wrong)

Reply:
Yeaah, I remember I already posted somewhere that our forking is too aggressive: https://en.wikipedia.org/wiki/Random_recursive_tree